internal/xds/translator/cluster.go

```diff
@@ -144,6 +145,16 @@ func buildXdsCluster(args *xdsClusterArgs) *clusterv3.Cluster {
 		cluster.ClusterDiscoveryType = &clusterv3.Cluster_Type{Type: clusterv3.Cluster_STRICT_DNS}
 		cluster.DnsRefreshRate = durationpb.New(30 * time.Second)
 		cluster.RespectDnsTtl = true
+		if args.dns != nil {
+			if args.dns.DNSRefreshRate != nil {
+				if args.dns.DNSRefreshRate.Duration > 0 {
+					cluster.DnsRefreshRate = durationpb.New(args.dns.DNSRefreshRate.Duration)
+				}
+				if args.dns.RespectDNSTTL != nil {
```

Comment:
should this live outside the the `args.dns.DNSRefreshRate` nil check, or does envoy require `DNSRefreshRate` to be set for `RespectDNSTTL` to be set?

Reply:
we default as per above:
	cluster.DnsRefreshRate = durationpb.New(30 * time.Second)
	cluster.RespectDnsTtl = true